### PR TITLE
[enhancement](load) improve error msg for load when cancelled by mem gc

### PR DESCRIPTION
**Pull request by freemandealer**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by freemandealer** [comment]
run buildall

**Comment by freemandealer** [comment]
run buildall

File: be/src/runtime/runtime_state.h [hunk 1]
```diff
@@ -536,6 +538,7 @@ class RuntimeState {
 
     // if true, execution should stop with a CANCELLED status
     std::atomic<bool> _is_cancelled;
+    std::string _cancel_reason;
```
Comment (dataroaring) [inline be/src/runtime/runtime_state.h:541]: please check thread safe.

**Review by dataroaring — CHANGES_REQUESTED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**